### PR TITLE
feat!: add new grays

### PR DESCRIPTION
**Pull request by marionnegp**
- Updated gray crayon values, names, and descriptions
- Changed semantic tokens that use gray crayon tokens

@coreyvickery, I used the Red Hat UI Color Usage Reference spreadsheet to map some of the semantic grays. Let me know if there are any that look incorrect or that you'd prefer to change.

### CODE REVIEW

File: tokens/color/crayon/gray.yaml [hunk 1]
```diff
@@ -7,55 +13,59 @@ color:
...
+    '80':
+      $value: '#1F1F1F'
+      $description: Primary surface (dark theme)
       attributes:
         type: gray
-    900:
+    '90':
       $value: '#151515'
-      $description: Primary surface (dark theme) or primary text (light theme)
+      $description: Primary canvas (dark theme) or primary text (light theme)
```
Comment (bennypowers) [inline tokens/color/crayon/gray.yaml:63]: we use surface above but canvas here?

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: tokens/color/crayon/gray.yaml (hunk 1 above)
Comment (coreyvickery) [reply tokens/color/crayon/gray.yaml:63]: @marionnegp `white` and `gray-90` can be applied to both canvas and surface use cases.

**Review by bennypowers — CHANGES_REQUESTED** [review]
Needs a changeset, with the `major` release level. Please add before/after snippets and instructions on how to use the new tokens

**Comment by marionnegp** [comment]
@coreyvickery, can you take a final look at the updated tokens? I also want to make sure that the descriptions are accurate.

**Comment by marionnegp** [comment]
@bennypowers, I can remove the update to 11ty.cjs. That was there to get the deploy preview to show properly. 

Also, is there a way to move the Canvas section above the Surface section? Adding a number to `order:` in the .yaml file didn't seem to do anything.

**Review by bennypowers — APPROVED** [review]

**Pull request merged**